src/ESPEasy-Globals.h

```diff
@@ -295,7 +295,7 @@
 #define PLUGIN_EXTRACONFIGVAR_MAX          16
 #define CPLUGIN_MAX                        16
 #define NPLUGIN_MAX                         4
-#define UNIT_MAX                           32 // Only relevant for UDP unicast message 'sweeps' and the nodelist.
+#define UNIT_MAX                           64 // Only relevant for UDP unicast message 'sweeps' and the nodelist.
```

Comment:
Do not just increase these numbers.
They are used in array definitions to mark their size. And those definitions are not just integers or booleans, but rather large structs. 
This one is about 16 bytes in size and a lot more when the name is set.
So setting this to 64 will increase minimal memory usage of this member to 1 kByte.

See:
https://github.com/letscontrolit/ESPEasy/blob/caee2892c42d723ff67fbb7bfd704d3d933a9a74/src/ESPEasy-Globals.h#L1233-L1245

We should use something like std::map to store the found nodes. Then the maximum number is not set and all kind of other issues will also disappear.